### PR TITLE
[docs] Change EA for read committed stable pages 

### PR DESCRIPTION
**Pull request by aishwarya24**
Slack requests:
- https://yugabyte.slack.com/archives/CDQB8MEAU/p1710356467914849
- https://yugabyte.slack.com/archives/CDQB8MEAU/p1710362588918629

### CODE REVIEW

File: docs/content/stable/develop/learn/transactions/acid-transactions-ysql.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ YugabyteDB supports three kinds of isolation levels to support different applica
...
+| [Read Committed {{<badge/ea>}}](../../../../explore/transactions/isolation-levels/#read-committed-isolation) | Each statement of the transaction sees the latest data committed by any concurrent transaction just before the execution of the statement. If another transaction has modified a row related to the current transaction, the current transaction waits for the other transaction to commit or rollback its changes. {{<note>}} The server internally waits and retries on conflicts, so applications [need not retry](../transactions-retries-ysql#automatic-retries) on serialization failures.{{</note>}} |
```
Comment (ddhodge) [inline docs/content/stable/develop/learn/transactions/acid-transactions-ysql.md:75]: Think the badge should be outside of link

File: docs/content/preview/yugabyte-platform/administer-yugabyte-platform/high-availability.md [hunk 2]
```diff
@@ -72,7 +72,7 @@ You can configure the active instance as follows:
 
     - If you installed YBA using [Replicated](../../install-yugabyte-platform/install-software/replicated/), locate the CA certificate from the path `/var/lib/replicated/secrets/ca.crt` on the YBA instance.
 
-1. Add the active instance CA certificate to the [YugabyteDB Anywhere trust store](../../security/enable-encryption-in-transit/#add-certificates-to-your-trust-store) of the active instance.
+1. Add the active instance CA certificate to the [YugabyteDB Anywhere trust store](../../security/enable-encryption-in-transit/#add-certificates-to-your-trust-store) of the standby instance.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/administer-yugabyte-platform/high-availability.md:75]: Unrelated - are you sure? If true, then do it like the following section, eg

1. Switch to the standby instance.
1. Add the active instance root certificate to the [YugabyteDB Anywhere trust store](../../security/enable-encryption-in-transit/#add-certificates-to-your-trust-store) on the standby instance. This allows the standby to connect to the active instance.

File: docs/content/stable/yugabyte-platform/administer-yugabyte-platform/high-availability.md [hunk 3]
```diff
@@ -69,7 +69,7 @@ You can configure the active instance as follows:
 
     - If you installed YBA using [Replicated](../../install-yugabyte-platform/install-software/replicated/), locate the CA certificate from the path `/var/lib/replicated/secrets/ca.crt` on the YBA instance.
 
-1. Add the active instance CA certificate to the [YugabyteDB Anywhere trust store](../../security/enable-encryption-in-transit/#add-certificates-to-your-trust-store) of the active instance.
+1. Add the active instance CA certificate to the [YugabyteDB Anywhere trust store](../../security/enable-encryption-in-transit/#add-certificates-to-your-trust-store) of the standby instance.
```
Comment (ddhodge) [inline docs/content/stable/yugabyte-platform/administer-yugabyte-platform/high-availability.md:72]: see above

File: docs/content/preview/yugabyte-platform/administer-yugabyte-platform/high-availability.md (hunk 2 above)
Comment (aishwarya24) [reply docs/content/preview/yugabyte-platform/administer-yugabyte-platform/high-availability.md]: I will verify this and create a different PR if needed. @ddhodge . For now I have reverted the change. So This PR has only the EA badge changes.

**Review by ddhodge — APPROVED** [review]
LGTM copy to preview?

**Pull request merged**